### DISCUSSION TITLE
MRTK3 Preview 17 Release

### DISCUSSION THREAD

**Discussion by AMollis** [microsoft/MixedRealityToolkit-Unity#11663] [Releases]
![MRTK3 Release Image for_v3.0.0-pre.17](https://github.com/microsoft/MixedRealityToolkit-Unity/assets/36461279/b64d831b-bca1-47f2-b493-9b8c841962f1)

Today the MRTK team is happy to announce the pre.17 release for MRTK3, one step closer to MRTK3 GA! This release includes some additional features. We've added resize and rotate cursors when interacting with bounding boxes, a spatial mouse interactor, and touch scrolling.  We've also fixed a variety of bugs.

As always, we welcome your feedback on these changes and look forward to reaching MRTK3 GA soon.

# Changelog for MRTK3 pre.17

## [pre.17] - 2023-6-28 - [`702ee31`](https://github.com/microsoft/MixedRealityToolkit-Unity/commit/702ee31f9a34382d5ce1a444a8bba07405a63e3c)

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). The content is mirrored at [the changelog page of the Microsoft docs website](https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-overview/changelogs/mrtk3-pre-17-changelog), for now.

### Added

- Added scale and rotate cursor ([#11603](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11603))
- Added spatial mouse (experimental) ([#11643](https://github.com/microsoft/MixedRealityToolkit-Unity/commit/64922a342b7812f218dc4375c37f6f25ca7dfac1))
- Added touch and far ray list scrolling (experimental) ([#11642](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11642))
- Added missing `<summary>` tag documentation to public classes and members. ([#11614](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11614), [#11634](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11634), [#11622](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11622), [#11654](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11654))

### Changed

- Updated GraphicsTools package to 0.5.12 in samples and package dependencies ([#11638](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11638))
- Updated MRTK3's min Unity input system to 1.5.1 so to avoid stack overflows caused by 1.5.0 ([#11609](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11609))
- Marked `NonNativeKeyboard`, `VirtualizedScrollRectList`, accessibility types, and data binding types as experimental ([#11630](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11630))
- Updated material and texture serializations to Unity 2021 ([#11579](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11579))
- Changed `XRSubsystemHelpers` to use interfaces for MRTK subsystems ([#11641](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11641) and [#11651](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11651))

### Fixed

- Fixed z-fighting in MRTK3's non native keyboard prefab ([#11658](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11658))
- Fixed z-Fighting with MRTK3 canvas toggle buttons ([#11610](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11610))
- Fixed Gaze Input so that it falls back to head gaze when the eye-gaze device loses tracking ([#11601](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11601))
- Fixed NonNativeKeyboard Sprite References ([#11608](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11608))
- Fixed MRTK cross package dependencies ([#11623](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11623))

**Comment by Victor-x1** [comment]
Is MRTK3 still being work on? I saw last commit to the MRTK3 branch was 2 weeks ago . :(

**Reply by AMollis** [reply]
@Victor-x1 ... stay tuned for an exciting update on Monday!
